contracts/test/_lib/testContract.js

```diff
@@ -0,0 +1,22 @@
+/* global assert, web3 */
+/** ensures use of pre-configured web3 if provided */
+let web3New
+if (typeof web3 === 'undefined') {
```

Comment:
can we abstract this logic for `web3New` into a common file and just expose one web3 thing for all these tests to use? also i see references to both web3.<function> and web3New.<function>. we should move it all to this wrapper, whatever you want to call it. you can keep it web3New

Reply:
👍